### PR TITLE
mail-signup + flood extents + no dupl pop. + etc

### PR DESCRIPTION
**Pull request by jannisvisser**

### CODE REVIEW

File: public/modules/dashboards/controllers/fbf.client.controller.js [hunk 1]
```diff
@@ -1373,6 +1377,26 @@ angular.module("dashboards").controller("FbfController", [
...
+      $scope.signup_mailing = function() {
+        $("#MailChimpModal").modal("show");
+      };
+      $scope.close_signup = function() {
+        $("#MailChimpModal").modal("hide");
+      };
+
+      $scope.join_whatsapp = function() {
+        $scope.whatsapp_link =
+          "https://chat.whatsapp.com/FH2hls1iOeNJF844L4cOjr";
```
Comment (elwinschmitz) [inline public/modules/dashboards/controllers/fbf.client.controller.js]: We should make values like this configurable 'from outside'.
So the same code can be used for multiple countries/groups/situations etc..

File: public/modules/dashboards/config/translations.client.config.js [hunk 2]
```diff
@@ -69,6 +69,9 @@ var static_en = {
   draft_warning: "",
   draft: "",
   export_json: "Export (JSON)",
+  export_pdf: "Export (PDF)",
+  signup_mailing: "Sign up to mailing list",
+  join_whatsapp: "Join Whatsapp group",
```
Comment (elwinschmitz) [inline public/modules/dashboards/config/translations.client.config.js]:
```suggestion
  join_whatsapp: "Join WhatsApp group",
```

File: public/modules/dashboards/views/fbf.client.view.html [hunk 3]
```diff
@@ -222,7 +224,7 @@
                 class="col col-md-5 col-sm-5 col-xs-5 general-component-label ng-binding ng-scope"
                 ng-click="change_indicator('fc_trigger')"
               >
-                Affected {{ subtype_selection }}
+                Exposed {{ subtype_selection }}
```
Comment (elwinschmitz) [inline public/modules/dashboards/views/fbf.client.view.html]: These string(s) can't be translated

File: public/modules/dashboards/views/fbf.client.view.html [hunk 4]
```diff
@@ -259,7 +261,7 @@
                 class="col col-md-5 col-sm-5 col-xs-5 general-component-label ng-binding ng-scope"
                 ng-click="change_indicator('population_affected')"
               >
-                Affected Population
+                Exposed Population
```
Comment (elwinschmitz) [inline public/modules/dashboards/views/fbf.client.view.html]: These string(s) can't be translated.

File: public/modules/dashboards/views/fbf.client.view.html [hunk 5]
```diff
@@ -800,4 +829,84 @@ <h2 class="h5 modal-title">{{ "warning" | translate }}</h2>
...
+              background: #fff;
+              clear: left;
+              font: 14px Helvetica, Arial, sans-serif;
+            }
+            /* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
+              We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
+          </style>
+          <div id="mc_embed_signup">
+            <form
+              action="https://global.us20.list-manage.com/subscribe/post?u=acdf2ad027e3073babce9067d&amp;id=f3a585f6ba"
```
Comment (elwinschmitz) [inline public/modules/dashboards/views/fbf.client.view.html]: This URL also shouldn't be hard-coded, as it contains 'our' account-number/id.

File: public/modules/dashboards/views/fbf.client.view.html [hunk 6]
```diff
@@ -800,4 +829,84 @@ <h2 class="h5 modal-title">{{ "warning" | translate }}</h2>
...
+                  required
+                />
+                <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                <div
+                  style="position: absolute; left: -5000px;"
+                  aria-hidden="true"
+                >
+                  <input
+                    type="text"
+                    name="b_acdf2ad027e3073babce9067d_f3a585f6ba"
```
Comment (elwinschmitz) [inline public/modules/dashboards/views/fbf.client.view.html]: This string also shouldn't be hard-coded, as it contains a specific value to 'our' account.

File: public/modules/dashboards/views/fbf.client.view.html [hunk 7]
```diff
@@ -800,4 +829,84 @@ <h2 class="h5 modal-title">{{ "warning" | translate }}</h2>
...
+          <button type="button" class="close" data-dismiss="modal">
+            <i class="fa fa-times-circle-o" aria-hidden="true"></i>
+          </button>
+          <h2 class="h5 modal-title">
+            {{ "Sign up to mailing list" | translate }}
+          </h2>
+        </div>
+        <div class="modal-body">
+          <link
+            href="//cdn-images.mailchimp.com/embedcode/slim-10_7.css"
```
Comment (elwinschmitz) [inline public/modules/dashboards/views/fbf.client.view.html]: I don't think we would need all the external styles loaded from MailChimp.
We could re-use the styles for form-fields we already have from Bootstrap.

File: public/modules/dashboards/data/static_en.json [hunk 8]
```diff
@@ -12,6 +12,7 @@
   "share_url": "Share full current settings (URL)",
   "export_csv": "Export (CSV)",
   "export_geojson": "Export (GEOJSON)",
+  "signup_mailing": "Sign up to mailing list",
```
Comment (elwinschmitz) [inline public/modules/dashboards/data/static_en.json]: This doesn't include the "Export (PDF)" or "Join WhatsApp group" values; shouldn't all strings align between these 2 files? Or can we remove 1 of the 2?

File: public/modules/dashboards/controllers/fbf.client.controller.js (hunk 1 above)
Comment (jannisvisser) [reply public/modules/dashboards/controllers/fbf.client.controller.js]: Thanks @elwinschmitz! went through these comments and made adjustments :)

**Pull request merged**